lib/actions/slack/slack_client_manager.js

```diff
@@ -8,10 +8,17 @@ exports.PLACEHOLDER_WORKSPACE = "any";
 exports.MULTI_WORKSPACE_SUPPORTED_VERSION = "7.3.0";
 const isSupportMultiWorkspaces = (request) => request.lookerVersion && semver.gte(request.lookerVersion, exports.MULTI_WORKSPACE_SUPPORTED_VERSION);
 exports.isSupportMultiWorkspaces = isSupportMultiWorkspaces;
-const makeSlackClient = (token) => new web_api_1.WebClient(token);
+const makeSlackClient = (token, disableRetries = false) => {
+    if (disableRetries) {
+        return new web_api_1.WebClient(token, { retryConfig: { retries: 0 } });
+    }
+    else {
+        return new web_api_1.WebClient(token);
+    }
+};
 exports.makeSlackClient = makeSlackClient;
 class SlackClientManager {
-    constructor(request) {
+    constructor(request, isForm = false) {
```

Comment:
Nit: Perhaps just make this explicit that this controls retries. s/isForm/disableRetries. Then in slack action, with new slack client it's obvious at that point you're making the choice to enable retries for forms.

Reply:
ack